README.md

```diff
@@ -58,6 +64,13 @@ Reference contracts that the Helix system uses to make everything trustless.
 Everything comes together here. This is a simulated environment where you can see how to wire
 everything up and even tweak with some parameters for your dApp.
 
-## License
+### [`notebooks`](packages/examples/notebooks/)

```

Comment:
I put these in `examples` by mistake (removing in #81), permanent home is packages/bootleg-tokens.

I would think we should reference it under the `bootleg token` header rather than it's own package. If we want to make it it's own package, I'd be good with that too.

Reply:
That's fine to have it under the token. Makes sense.